src/app/rundown-view/components/sticky-segment/sticky-segment.component.ts

```diff
@@ -0,0 +1,50 @@
+import { AfterViewInit, Component, ElementRef, Input, OnDestroy } from '@angular/core'
+import { Segment } from '../../../core/models/segment'
+
+@Component({
+  selector: 'sofie-sticky-segment',
+  templateUrl: './sticky-segment.component.html',
+  styleUrls: ['./sticky-segment.component.scss'],
+})
+export class StickySegmentComponent implements AfterViewInit, OnDestroy {
```

Comment:
Does this really need to be a new component?
Is basically just the SegmentComponent with a CSS class added to it.

Reply:
It keeps the original segment component cleaner and makes it easier to extend the functionality of the sticky segment without altering the original segment component – e.g. if we want to add a button for scrolling it into view or something else.

Reply:
My biggest issue is that almost all the input variables (which there are quite a few of) is just parse-through...

Reply:
I see the point Lindved is making about everything being passed through, but i think it makes sense to have some general pattern we implement to avoid growing components further, especially if we can argue that this makes for a new "version" of a component.